BREEZE.rst

```diff
@@ -347,74 +329,105 @@ can check whether your problem is fixed.
 4. `Clean up Docker images <#cleaning-up-the-images>`_.
 5. Restart your Docker Engine and try again.
 6. Restart your machine and try again.
-7. Remove and re-install Docker CE and try again.
+7. Re-install Docker CE and try again.
 
 In case the problems are not solved, you can set the VERBOSE variable to "true" (``export VERBOSE="true"``),
 rerun the failed command, copy-and-paste the output from your terminal to the _`Airflow Slack <https://apache-airflow-slack.herokuapp.com/>`_  #troubleshooting channel and add the problem description.
 
 
-Using Breeze for Other Tasks
-============================
+Testing in Breeze
+=================
+
+Running Unit Tests in Airflow Breeze
+-------------------------------
```

Comment:
One comment here - in .rst the ------ lines length below header must much the header length :)